iocore/net/I_NetVConnection.h

```diff
@@ -566,6 +573,21 @@ class NetVConnection : public VConnection
     is_transparent = state;
   }
 
+  /// Set the context and return context value
+  NetVConnectionContext_t
+  set_context(NetVConnectionContext_t context)
```

Comment:
The return value tells you whether it sets successful or not.
Yes, the context is only set once and will not change.
- Due to the VC is create by Allocator, It can not be set in the constructor.
- Assert here and set return type to void. ( update later )